### DISCUSSION TITLE
⏳ 🔗 Loop builder for TABS

### DISCUSSION THREAD

**Discussion by elasticeye** [elementor/elementor#21245] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

An easy way to populate a tab widget using the loop builder.

### Describe the solution you'd like

Connect the loop builder to the tabs widget.

### Describe alternatives you've considered

Crocoblock have an option.

**Comment by fabigrafo** [comment]
What you mean? You can put a loop grid or slider in a nested tab widget and you can put a nested tab widget in the loop builder? What do you want more?

**Reply by elasticeye** [reply]
Hmm, I did not know that. So I can create 5 posts, and populate 5 different tabs in the tab widget using the loop builder?

**Reply by loucasl** [reply]
I think he means to dynamically generate tabs, also with a custom loop inside

So if you have 5 posts, 5 tabs, 10 posts, 10 tabs etc

**Reply by elasticeye** [reply]
> I think he means to dynamically generate tabs, also with a custom loop inside
> 
> So if you have 5 posts, 5 tabs, 10 posts, 10 tabs etc

Yes, exactly.

**Reply by fabigrafo** [reply]
Okay, yes you can display in each tab 1 loop grid item if you want. If you mean automatically like a filter bar, you must wait for the loop builder filter bar.

**Comment by elasticeye** [comment]
This should be implemented on ALL repeater widgets, like tabs, accordian etc...

**Reply by fabigrafo** [reply]
> This should be implemented on ALL repeater widgets, like tabs, accordian etc...

It will in future, TABs was only the start.

**Reply by elasticeye** [reply]
> > This should be implemented on ALL repeater widgets, like tabs, accordian etc...
> 
> It will in future, TABs was only the start.

Yes, nested tabs is here. But I don't think you can connect a loop to a tab, and auto populate the tabs based on x number of posts.

**Comment by ayoubkhan558** [comment]
+1

**Comment by marcieee73** [comment]
+1
i would love to have a loop tabs. I tried to insert statically loops in nested tabs. the page became so heavy that it was impossible to work in it. I don't know if making dynamic tabs would improve the performance, anyway it would very useful to have this feature.
